--- _include/pthread_port.h.orig	2024-03-29 14:06:17
+++ _include/pthread_port.h	2024-04-04 06:30:32
@@ -61,29 +61,9 @@
 #include <sys/time.h>
 #include <time.h>
 
-#if defined(__MACH__) && !defined(CLOCK_REALTIME)
-#include <mach/clock.h>
-#include <mach/mach.h>
 #ifndef CLOCK_REALTIME
 #define CLOCK_REALTIME 0
 #endif
-static inline void clock_gettime(clk_id_unused, struct timespec *ts)
-{
-  clock_serv_t cclock;
-  mach_timespec_t mts;
-  host_get_clock_service(mach_host_self(), CALENDAR_CLOCK, &cclock);
-  clock_get_time(cclock, &mts);
-  mach_port_deallocate(mach_task_self(), cclock);
-  ts->tv_sec = mts.tv_sec;
-  ts->tv_nsec = mts.tv_nsec;
-}
-#elif !defined(HAVE_CLOCK_GETTIME)
-static inline void clock_gettime(clk_id_unused, struct timespec *ts)
-{
-  ts->tv_sec = time(0);
-  ts->tv_nsec = 0;
-}
-#endif
 
 // FREE
 static void __attribute__((unused)) free_if(void *ptr) { free(*(void **)ptr); }
