diff --git a/Makefile.inc.in b/Makefile.inc.in
index d9fc1a953..cadcd2286 100644
--- a/Makefile.inc.in
+++ b/Makefile.inc.in
@@ -103,9 +103,6 @@ HELP_DESCRIPTION = This is MDSplus build system, documented targets follow:
 ## docker build targets
 @AX_DOCKER_BUILD_TARGETS@
 
-## git release tags
-@AX_GIT_REVISION_TARGETS@
-
 
 .PHONY: reconfigure
 reconfigure: ##@miscellaneous rerun configure script again using last arguments passed
